cfme/v2v/__init__.py

```diff
@@ -0,0 +1,37 @@
+from navmazing import NavigateToSibling
+from widgetastic.widget import Text
+
+from cfme.base import Server
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigator
+from cfme.utils.version import Version
+from cfme.utils.version import VersionPicker
+
+
+class MigrationView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        nav_menu = VersionPicker(
+            {
+                Version.lowest(): ["Compute", "Migration"],
+                "5.10": ["Compute", "Migration", "Migration Plans"],
+            }
+        )
+
+        return self.logged_in_as_current_user and self.navigation.currently_selected == nav_menu
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+@navigator.register(Server)
+class Migration(CFMENavigateStep):
+    VIEW = MigrationView
+    prerequisite = NavigateToSibling("LoggedIn")
+
+    def step(self):
+        self.view.navigation.select("Compute", "Migration")
```

Comment:
Is this correct for 5.10? There is not Compute -> Migration page. Looks like this is only valid for 5.9. Is this a work around for covering the case in 5.9 but the pages changing in 5.10? It seems that way by what is being check in the `in_explorer()` method.